cmd/mycelo/main.go

```diff
@@ -72,12 +74,23 @@ var gethPathFlag = cli.StringFlag{
 	Usage: "Path to geth binary",
 }
 
+var gethExtraFlagsFlag = cli.StringFlag{
+	Name:  "extraFlags",
```

Comment:
Maybe name it `extraflags` to be consistent with `maxpending` and `blockgaslimit`